src/main/java/Event.java

```diff
@@ -0,0 +1,25 @@
+public class Event extends Task {
+
+    // added attribute
+    private String at;
+
+    public Event(String desc, String at) {
+        super(desc);
+        setAt(at);
+    }
+
+    public void setAt(String at) {
+        this.at = at;
+    }
+
+    // override method
```

Comment:
Should this be @override instead? If you think so too, then it would be applicable to the other overridden methods that you have!